src/m365/tenant/commands/people/people-profilecardproperty-list.ts

```diff
@@ -0,0 +1,78 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import GraphCommand from '../../../base/GraphCommand.js';
+import request, { CliRequestOptions } from '../../../../request.js';
+import { ProfileCardProperty } from './profileCardProperties.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: GlobalOptions;
+}
+
+class TenantPeopleProfileCardPropertyListCommand extends GraphCommand {
+  public get name(): string {
+    return commands.PEOPLE_PROFILECARDPROPERTY_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists all profile card properties';
+  }
+
+  constructor() {
+    super();
+  }
+
+  public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
+    try {
+      if (this.verbose) {
+        await logger.logToStderr(`Lists all profile card properties...`);
+      }
+
+      const requestOptions: CliRequestOptions = {
+        url: `${this.resource}/v1.0/admin/people/profileCardProperties`,
+        headers: {
+          accept: 'application/json;odata.metadata=none'
+        },
+        responseType: 'json'
+      };
+
+      const result = await request.get<ProfileCardProperty[]>(requestOptions);
+      let output: any = result;
+
+      if (args.options.output && args.options.output !== 'json') {
+        output.value.sort((p1: ProfileCardProperty, p2: ProfileCardProperty) => {
+          const firstPropertyAnnotationLocalizationsCount: number = p1.annotations[0]?.localizations?.length ?? 0;
+          const secondPropertyAnnotationLocalizationsCount: number = p2.annotations[0]?.localizations?.length ?? 0;
+          if (firstPropertyAnnotationLocalizationsCount < secondPropertyAnnotationLocalizationsCount) {
+            return 1;
+          }
+          if (firstPropertyAnnotationLocalizationsCount > secondPropertyAnnotationLocalizationsCount) {
+            return -1;
+          }
+          return 0;
+        });
+
+        output = output.value.map((p: ProfileCardProperty) => {
+          const propertyAnnotations = p.annotations[0]?.localizations?.map((l) => {
+            return { ['displayName ' + l.languageTag]: l.displayName };
+          }) ?? [];
+
+          const propertyAnnotationsObject = Object.assign({}, ...propertyAnnotations);
+
+          return {
+            directoryPropertyName: p.directoryPropertyName,
+            displayName: p.annotations[0]?.displayName ?? '',
```

Comment:
I'm not sure we should add an empty displayName if none is coming from the API. Lets not add the property in that case.

Reply:
ok good point. I removed it and cleared the output in the spec